odysseus/odysseus_tree/board/tpu/plugin_user_property_timestamp.c

```diff
@@ -0,0 +1,48 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <sys/time.h>
+
+
+#include "nanomq/plugin.h"
+
+/*
+ * How to compile:
+ * gcc -I../ -fPIC -shared plugin_user_property.c -o plugin_user_property.so
+ */
+
+
+int cb(void *data)
```

Comment:
Can you rename this to something more verbose, especially since we can manually link the function here? `cb` is very generic even if it is a small file.

On the same vein, can we rename `data` to something more verbose about what data we are getting?

Reply:
some of that is dictacted by the source code as this is passed directly to the publisher as a hook.  https://github.com/nanomq/nanomq/blob/d271182eb11f8a9393f051f359e750463a090d63/nanomq/plugin/plugin.c#L44